app.py

```diff
@@ -19,12 +19,18 @@
     load_stats_data,
     transform_loans_data,
     transform_main_chart_data,
+    ProtocolColors,
 )
 
+
+ZKLEND = "zkLend"
```

Comment:
you have duplicated ZKLEND, NOSTRA_ALPHA,  NOSTRA_MAINNET and PROTOCOL_NAMES please, use it from `chart_utils.py`